plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/project/DataflowProjectCreator.java

```diff
@@ -194,31 +195,34 @@ public void run(IProgressMonitor monitor)
             mavenGroupId, mavenArtifactId, "0.0.1-SNAPSHOT", packageString, archetypeProperties,
             projectImportConfiguration, progress.newChild(4));
         break;
-      } catch (CoreException e) {
-        failures.add(e);
+      } catch (CoreException ex) {
+        // todo should we use a MultiStatus here?
+        failures.add(ex);
       }
     }
-    if (projects.isEmpty()) {
+    if (!failures.isEmpty()) {
```

Comment:
This change is not correct.  The code above is tries a set of archetype versions.  Providing the last attempt worked (i.e., the project was created) then all is well.